src/config/app-schema.js

```diff
@@ -0,0 +1,29 @@
+// istanbul ignore file
+import Joi from 'joi';
```

Comment:
This all seems request specific and not config. The original purpose of config here was to ensure we have only one place where `process.env` values are processed. Not saying it's wrong but it doesn't seem to fit quite right or our understanding of how we're architecting this is mis-aligned.

Reply:
Hmm maybe but to me modern applications often treat configuration as more than just process.env variables, combining defaults, validations, and derivations in a central place.

I could move some of the `request` centric config items to `request-config.js` if it bothers you. We can always discuss tomorrow. I think the joi schema validation is useful though.

We can always discuss tomorrow and align. Maybe we can even document in README somewhere the role of `config` in our architecture so that we know where the line is drawn between pure configuration, runtime behavior, and application logic. WDYT?